pages/linux/goldeneye.md

```diff
@@ -0,0 +1,24 @@
+# goldeneye
```

Comment:
Page name should match the command name, please rename the file to `goldeneye.py.md` too
```suggestion
# goldeneye.py
```